app/views/spree/admin/shared/sub_menu/_integrations.html.erb

```diff
@@ -1,3 +1,5 @@
 <ul id="sidebar-integrations" class="border-top bg-white collapse nav nav-pills nav-stacked" data-hook="admin_integrations">
   <%= tab :webhooks_subscribers, match_path: '/webhooks_subscribers', label: 'Webhooks' %>
+  <%= tab :payment_methods, match_path: '/payment_methods', label: 'Payment Methods' %>
```

Comment:
@aleksandarpetrushev please also copy-over `can :manage...` as this is required for proper cancancan integration